nixos/modules/config/qt5.nix

```diff
@@ -9,8 +9,18 @@ let
   isQGnome = cfg.platformTheme == "gnome" && builtins.elem cfg.style ["adwaita" "adwaita-dark"];
   isQtStyle = cfg.platformTheme == "gtk2" && !(builtins.elem cfg.style ["adwaita" "adwaita-dark"]);
 
-  packages = if isQGnome then [ pkgs.qgnomeplatform pkgs.adwaita-qt ]
-    else if isQtStyle then [ pkgs.libsForQt5.qtstyleplugins ]
+  packages = if isQGnome then [
```

Comment:
I don't know what this config does, but this seems a bit like using a hammer?

Reply:
TBH I don't use it as well. I think it just sets some env vars and adds packages to the system path, not sure how exactly that does what it should, but @worldofpeace is the author of it so they should know better.